node/pkg/watchers/sui/watcher.go

```diff
@@ -306,14 +314,14 @@ func (e *Watcher) Run(ctx context.Context) error {
 
 			resp, err := http.Post(e.suiRPC, "application/json", strings.NewReader(buf))
 			if err != nil {
-				logger.Error(e.suiRPC, zap.Error(err))
+				logger.Error(fmt.Sprintf("http.Post to %s", e.suiRPC), zap.Error(err))
```

Comment:
Normally this would be something like:
`logger.Error("failed to post http request", zap.String("suiRPC", e.suiRPC), zap.Error(err))`
Same comment on the error logs below.

Reply:
this good?